plutus-scb/src/Cardano/Metadata/Types.hs

```diff
@@ -272,6 +273,13 @@ toPropertyKey (Name _ _)        = PropertyKey "name"
 toPropertyKey (Description _ _) = PropertyKey "description"
 toPropertyKey (Other name _ _)  = PropertyKey name
 
+newtype Query = QuerySubjects (Set Subject)
```

Comment:
This is "all properties for all these subjects", right? I wonder if we also want "specified set of properties for all of a set of subjects"?

Reply:
Yeah, that's a good idea. And call me crazy, but I added tests for it too.

Reply:
Done. :-)